### PR TITLE
Allowed custom lines to be added to the rules file using the 'custom_…

### PR DESCRIPTION
**Pull request by triforce**
…cfg' configuration parameter.

### CODE REVIEW

**Comment by juniorsysadmin** [comment]
@triforce This PR needs a rebase. Are you still interested in merging this?

**Comment by triforce** [comment]
Yes please. Am I rebasing to master?

**Comment by juniorsysadmin** [comment]
> Yes please. Am I rebasing to master?

Rebasing on top of master. If you could add some tests, that would be great as well.

**Comment by triforce** [comment]
I'm not too familiar with the automated testing side of things for puppet but added something simple in.

**Comment by triforce** [comment]
Is there something else I need to do here?

File: manifests/rule.pp [hunk 1]
```diff
@@ -123,6 +124,7 @@
 define logrotate::rule(
   Pattern[/^[a-zA-Z0-9\._-]+$/] $rulename           = $title,
   Enum['present','absent'] $ensure                  = 'present',
+  Array[String] $custom_cfg                         = [],
```
Comment (bastelfreak) [inline manifests/rule.pp:127]:
```suggestion
  Array[String[1]] $custom_cfg                      = [],
```

**Comment by bastelfreak** [comment]
thanks for the patch!

**Pull request merged**